src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/ConfigReader.java

```diff
@@ -0,0 +1,22 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.deciders.Decider;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class ConfigReader {
+    public static HashMap<Decider, ArrayList<String>> getDeciderActionPriorityOrder() {
+        ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+        HashMap<Decider, ArrayList<String>> deciderActionPriorityOrder = null;
```

Comment:
Looks like the yml is read and created each time someone fetches action priority. How about parsing and storing it ones, during load/init?

Reply:
Yup. Updated in the latest version.